npm/vite-dev-server/cypress/e2e/vite-dev-server.cy.ts

```diff
@@ -44,22 +44,23 @@ describe('Config options', () => {
   })
 
   it('supports supportFile = false', () => {
-    cy.scaffoldProject('vite2.9.1-react')
-    cy.openProject('vite2.9.1-react', ['--config-file', 'cypress-vite-no-support.config.ts', '--component'])
+    cy.scaffoldProject('vite5.4.10-react')
+    cy.openProject('vite5.4.10-react', ['--config-file', 'cypress-vite-no-support.config.ts', '--component'])
     cy.startAppServer('component')
 
     cy.visitApp()
     cy.specsPageIsVisible()
     cy.contains('App.cy.jsx').click()
     cy.waitForSpecToFinish()
-    cy.get('.passed > .num').should('contain', 1)
+    // no support file means there is no mount function registered, so all tests should fail
+    cy.get('.failed > .num').should('contain', 2)
```

Comment:
honestly not sure why this ever passed initially, but I would argue this is the correct behavior since there is no mount function.